_source/logzio_collections/_technical-notes/chain-of-trust.md

```diff
@@ -0,0 +1,209 @@
+  Re-deploy using the instructions in
+  _[Ship Kubernetes metrics]({{site.baseurl}}/shipping/metrics-sources/kubernetes.html)_
+
+#### To replace the certificate file {#replace-the-cert-file}
+
+This covers instructions for any of the Beats shippers
+(such as Filebeat, Metricbeat, Winlogbeat, or Auditbeat)
+and Logstash over SSL.
+
+If you're managing your shipping app
+on a VM or another server,
+follow the steps below
+to overwrite the expiring certificate file
+with a new certificate file.
+
+The new certificate file contains both the old and the new certificates,
+allowing you to ship data without interruption when we update our listeners.
+
+<div class="tasklist">
+
+##### Download the Logz.io public certificates
+
+The new certificate file contains both the old and the new certificate,
+allowing you to ship data without interruption when we update our listeners.
+
+The commands below save the new certificate file
+to the location that we recommend in our docs and sample configurations.
+Double-check the file paths,
+just in case your team uses a different file location.
+{:.info-box.important}
+
+###### For Filebeat or Metricbeat (macOS or Linux)
+
+```shell
+sudo curl https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt --create-dirs -o /etc/pki/tls/certs/COMODORSADomainValidationSecureServerCA.crt
+```
+
+###### For Winlogbeat or Filebeat (Windows)
+
+Download the
+[Logz.io public certificate](https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt)
+
+* For **Winlogbeat**: Copy to `C:\ProgramData\Winlogbeat\COMODORSADomainValidationSecureServerCA.crt`
+* For **Filebeat**: Copy to `C:\ProgramData\Filebeat\Logzio.crt`
+
+###### For Logstash over SSL
+
+```shell
+sudo curl https://raw.githubusercontent.com/logzio/public-certificates/master/TrustExternalCARoot_and_USERTrustRSAAAACA.crt --create-dirs -o /usr/share/logstash/keys/TrustExternalCARoot.crt
+```
+
+##### Check your configuration
+
+Double-check your shipper's configuration file
+to make sure it's using the certificate at the location you downloaded it to.
+
+If the certificate location in the configuration doesn't match
+where you downloaded the file to,
+update the configuration or move the certificate.
+
+###### For a Beats family shipper
+
+The certificate is included
+in the `ssl.certificate_authorities` array
+in your configuration file.
+
+###### For Logstash over SSL
+
+The certificate is included
+in `output` > `lumberjack` > `ssl_certificate` setting
+in your configuration file.
+
+##### Restart your shipper
+
+Restart your shipper to load the new certificate.
+
+##### Test the new certificate
+
+Starting Monday, May 18,
+we’ll open up test listeners
+so you can confirm the new configurations will work
+after we switch to the new chain of trust.
+
+Test your new configuration
+by temporarily pointing your shipper to a testing URL for your region
```

Comment:
suggestion 
```
by temporarily pointing a local copy of the shipper to a testing URL for your region
```
they dont need to point all of their shippers, but only to test it on one local **or** remote server

Reply:
I'm going to leave it — I'd rather they made their own decisions about what to test.